datetime/time.go

```diff
@@ -361,6 +361,17 @@ func NewDateFromEpoch(epoch int64) Date {
 	}
 }
 
+// TimeFromDate returns a time.Time for a date
+func TimeFromDate(date Date) time.Time {
+	ts := DateFromEpoch(date.Epoch)
+	if ts.IsZero() {
+		return ts
+	}
+	// apply timezone
+	loc := time.FixedZone("pinpt", int(date.Offset*60))
+	return ts.In(loc)
```

Comment:
what in the world? the timezone will come back as "pinpt"

Reply:
far as I can tell its just a name i can make it blank, but I wanted to have some fun 🤷‍♀️

Reply:
i thiink you want this

```
loc := time.FixedZone(ts.Location().String(), int(date.Offset*60))
```